src/styles/blocks/container.scss

```diff
@@ -0,0 +1,32 @@
+.container {
+  display: grid;
+  grid-template-rows: repeat(2, $size);
+  grid-template-columns: repeat(3, minmax($size, 1fr));
+  grid-auto-flow: row;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+
+  @media (max-width: 750px) {
+    grid-template-rows: repeat(3, $size);
+    grid-template-columns: repeat(2, minmax($size, 1fr));
+  }
+
+  @media (max-width: 450px) {
```

Comment:
check your breakpoints https://github.com/mate-academy/layout_antisnake/tree/master/reference

Reply:
Marina, can't find bug in breakpoints(

Reply:
two columns should only appear on the width 600px and more, and three columns should only appear on the width 900px and more